src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -223,11 +223,17 @@ export class MdAutocompleteTrigger implements AfterContentInit, ControlValueAcce
     }
   }
 
-  _handleBlur(newlyFocusedTag: string): void {
+  /**
+   * Handles the input element losing focus. Note that this should be handled on `focusout`,
+   * instead of `blur`, because IE doesn't set the `relatedTarget` for blur events.
+   */
+  _handleFocusOut(event: FocusEvent): void {
+    const relatedTarget = event.relatedTarget as HTMLElement;
+
     this._onTouched();
 
-    // Only emit blur event if the new focus is *not* on an option.
-    if (newlyFocusedTag !== 'MD-OPTION') {
+    // Only emit blur event if the new focus is *not* on an element inside the panel.
+    if (relatedTarget && !this._overlayRef.overlayElement.contains(relatedTarget)) {
```

Comment:
This appears to break clicking outside the panel to close it in Chrome and Firefox. Can you fix?